might eventually be able to associate the version... (#255)
quicwg/ops-drafts: draft-ietf-quic-manageability.md

Initial packet with the same version implies acceptance of that version.

Negotiated version cannot be identified for flows for which a handshake is not

> observed, such as in the case of connection migration; however, these flows
> might eventually be associated with flows for which a version has been

Reviewer: ```suggestion
observed, such as in the case of connection migration; however, it might be
possible to associate a flow with a flow for which a version has been
```

Eventually implies an inevitability, which still implies that our linkability protection is ineffectual.

Author: If you only switch CID (and maybe one IP address) but keep sending data with the same speed to the same endpoint on the same destination port, I would actually say the linkability protection is quite ineffectual.
